### PR TITLE
[DOC] add Parameter estimate/beta to glossary

### PR DESCRIPTION
**Pull request by mekman**
Adding Parameter estimate/beta to glossary. No functional changes.

See #2913

### CODE REVIEW

File: doc/glossary.rst [hunk 1]
```diff
@@ -21,6 +21,9 @@ If you wish to add a missing term, please `create a new issue`_ or
     AUC
         `Area under the curve`_.
 
+    Beta
+        `Parameter estimate`_.
```
Comment (NicolasGensollen) [inline doc/glossary.rst:25]:
```suggestion
        See :term:`Parameter estimate`.
```

File: doc/glossary.rst [hunk 2]
```diff
@@ -127,6 +130,15 @@ If you wish to add a missing term, please `create a new issue`_ or
         `Neurovault`_ is a public repository of unthresholded statistical maps, 
         parcellations, and atlases of the human brain.
 
+    Parameter estimate
+        In the context of a `GLM`_, each `contrast`_ comparing rows in the
```
Comment (NicolasGensollen) [inline doc/glossary.rst:134]:
```suggestion
        In the context of a :term:`GLM`, each :term:`contrast` comparing rows in the 
```

File: doc/glossary.rst [hunk 3]
```diff
@@ -127,6 +130,15 @@ If you wish to add a missing term, please `create a new issue`_ or
         `Neurovault`_ is a public repository of unthresholded statistical maps, 
         parcellations, and atlases of the human brain.
 
+    Parameter estimate
+        In the context of a `GLM`_, each `contrast`_ comparing rows in the 
+        design matrix results in a parameter estimate (PE) that signifies how 
+        well the underlying model fits the data at each voxel. For statistical
```
Comment (NicolasGensollen) [inline doc/glossary.rst:136]:
```suggestion
        well the underlying model fits the data at each :term:`voxel`. For statistical 
```

File: doc/glossary.rst [hunk 4]
```diff
@@ -127,6 +130,15 @@ If you wish to add a missing term, please `create a new issue`_ or
         `Neurovault`_ is a public repository of unthresholded statistical maps, 
         parcellations, and atlases of the human brain.
 
+    Parameter estimate
+        In the context of a `GLM`_, each `contrast`_ comparing rows in the 
+        design matrix results in a parameter estimate (PE) that signifies how 
+        well the underlying model fits the data at each voxel. For statistical 
+        inferences the parameter estimate, sometimes also referred to as 
+        `beta`_, is commonly converted to either a t-, or z-statistic. In
```
Comment (NicolasGensollen) [inline doc/glossary.rst:138]:
```suggestion
        :term:`beta`, is commonly converted to either a t-, or z-statistic. In 
```

**Review by NicolasGensollen — CHANGES_REQUESTED** [review]
Thanks for opening this @mekman !
I left a few suggestions below. 
Note that if you want to reference a term in the glossary you need to use 
```
:term:`word_in_glossary`
```

**Comment by mekman** [comment]
Thank you @NicolasGensollen for pointing that out. Fixed now.

**Comment by tsalo** [comment]
Can you perhaps link to the glossary in the `compute_contrast` docstring and/or in examples as well?

**Comment by mekman** [comment]
Just a quick note, I will continue working on this by the end of August. Thanks for the feedback so far.

**Comment by NicolasGensollen** [comment]
Hi @mekman 
I think this was almost ready to be merged and only missing a few references to the new entry in the glossary as @tsalo suggested.
Let me know if I can help.

**Comment by tsalo** [comment]
@mekman are you able to finish this up? If not, would you mind if I made the final changes to get it ready to merge?

**Comment by tsalo** [comment]
I've made the necessary changes in #3330, so I will close this now.

**Pull request closed without merging**